templates/applicants.phtml

```diff
@@ -42,6 +43,7 @@
                     <td><a href="/admin/applicants/<?= $applicant->getId() ?>"><?= $applicant->getName(); ?></a></td>
                     <td><?php echo $applicant->getEmail(); ?></td>
                     <td><?php echo $applicant->getFormattedApplicationDate(); ?></td>
+                    <td><a href="/admin/applicants/edit/<?= $applicant->getId() ?>" class="edit-button">EDIT</a></td>
```

Comment:
Button should look like other buttons in the application. The colour can be different (and I do like the choice of the same pink as the iO Academy logo) but the border, corner-roundness, padding, etc. should be consistent.

This is a business application, so **no fun allowed**. The button should not have the pop-out hover effect - save that stuff for your portfolios. A hover color change with or without a CSS transition is fine, but make it transition to that light grey colour the other button goes to because the purple you chose is not in the colour scheme.

Reply:
Fred will do this issue

Reply:
Fun removed.  Buttons now fallen into line.